distributor/distributor.go

```diff
@@ -158,13 +173,14 @@ type sampleTracker struct {
 	succeeded  int32
 }
 
-// Append implements SampleAppender.
-func (d *Distributor) Append(ctx context.Context, samples []*model.Sample) error {
+// Push Implements cortex.IngesterServer
```

Comment:
Yeah, thats the idea.  Just this function really.  We do use the IngesterClient internally.